docs/dev/cli/virtual-usb/disconnect-session.md

```diff
@@ -4,15 +4,20 @@ title: "Virtual USB CLI: Disconnect Session"
 sidebar_label: Disconnect Session
 ---
 
-The `disconnect` command provides the server with your test session ID, then disconnects the real device from your local machine.
+The [`disconnect`](https://docs.saucelabs.com/mobile-apps/virtual-usb#close-test) command uses the `--sessionId` of a Virtual USB session started with the [`startSession`](dev/cli/virtual-usb/start-session) command to disconnect the real device from your local machine.
```

Comment:
@kimsauce 
This sentence is not correct

Reply:
Thank you for explaining these in your other comment. How do these sound? 

on the disconnect CLI page:
The [`disconnect`](https://docs.saucelabs.com/mobile-apps/virtual-usb#close-test) command closes out live Virtual USB test sessions that were started originally in Sauce Labs and linked to your Virtual USB client using the [`connect`](dev/cli/virtual-usb/connect-session) command.

on the deleteSession CLI page:
The [`deleteSession`](https://docs.saucelabs.com/mobile-apps/virtual-usb#close-test) command closes out Virtual USB tests session started in the command line interface using the [`startSession`](dev/cli/virtual-usb/start-session) command.

Reply:
@kimsauce 
Yep, that's better, thanks!